src/main.js

```diff
@@ -29,11 +29,11 @@ var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 
-var currentCover = null;
+var currentCover;
 
 //EVENT LISTENERS
 window.addEventListener("load", displayRandomHomePage);
-randomCoverButton.addEventListener("click", displayRandomImage);
+randomCoverButton.addEventListener("click", displayRandomHomePage);
```

Comment:
I would recommend changing the name of `displayRandomHomePage` to something that better fits what it is doing. Essentially, this function is just displaying a random poster, rather than the whole home page itself.

Reply:
Great idea! We will update to a displayRandomBook function.